src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -843,6 +819,8 @@ const DEFAULT_VALUES = {
   defaultScore: 0,
   dragMargin: 8,
   selectedDef: null,
+
+  numberOfBins: 32,
```

Comment:
I don't think the view components should individually set numberOfBins. They should get it from the provider.